homeassistant/components/islamic_prayer_times/sensor.py

```diff
@@ -48,7 +48,11 @@ def icon(self):
     @property
     def state(self):
         """Return the state of the sensor."""
-        return self.client.prayer_times_info.get(self.sensor_type).isoformat()
+        return (
+            self.client.prayer_times_info.get(self.sensor_type)
+            .astimezone(self.hass.config.time_zone)
```

Comment:
The result should still be UTC time. I think we may just want to attach the timezone?

https://docs.python.org/3/library/datetime.html#datetime.datetime.astimezone

Reply:
Actually the returned prayer_times are in local time which is calculated based on the submitted long/lat. The times were not in utc from the beginning. 
This will only attach the timezone taken from hass.config

Reply:
The state must be utc time.

Reply:
I am not sure what you mean. If for example the local time for the sensor is `01:00:00` and my timezone is `+03:00`, then state of the sensor should be `01:00:00+03:00` right?

Reply:
If it would be prayer time right now, the correct time would be returned by our utcnow util and then apply isoformatting to that:
https://github.com/home-assistant/core/blob/c71a7b901fea74cd2ce3b2a5305877f0cca2ae45/homeassistant/util/dt.py#L53-L55

Reply:
I changed the state to show as UTC. Please check